server/src/boards/boards.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Body, Controller, Get, Logger, Post } from '@nestjs/common'
+import { BoardEntity } from '../entities/Board'
+import { BoardsService } from './boards.service'
+
+@Controller('boards')
+export class BoardsController {
+  private readonly logger = new Logger(BoardsController.name)
+
+  constructor(private boardsService: BoardsService) {}
+
+  @Get()
+  getAllBoards(): Promise<BoardEntity[]> {
+    this.logger.log('GET /boards')
+
+    return this.boardsService.findAll()
+  }
+
```

Comment:
In java I would put a @Transactional annotation to make all request execute in a transaction. I should look up how to do that with this framework.